CIAC-11415 (#37778)
demisto/content: Packs/Absolute/ReleaseNotes/1_0_25.md

#### Integrations

##### Atlassian Confluence Cloud

> - Added the ability to fetch events in Cortex XSIAM.

Reviewer: ```suggestion
- Enhanced Cortex XSIAM with the capability to fetch events.
```
